README.md

```diff
@@ -95,6 +96,28 @@ Subsystem by running the following in PowerShell as an administrator
 Restart-Service LxssManager*
 ```
 
+### Local Development Login
+
+To log in to the system once locally deployed, simply enter a username in the designated field and click the "Login" button.
+
+#### Users & user types
+
+The following table lists the available usernames and their corresponding user types, as well as their associated permissions:
+
+| Username | Type of User | User Permission Scope |
+|:--------:|:-------------:|:----------------:|
+|developer | Developer user| developer |
+|researcher | Researcher user| researcher |
+|user04 | Pre-clinical-intake user| pre-clinical-intake |
+|user07 | Bioinformatics-section user| bioinformatics-section-user |
+
+Please note that the permissions listed in the table are subject to change.
+
+#### Login screenshots
+
```

Comment:
Can leave the screenshots added in the original section
```suggestion
```

Reply:
I have removed this subheading in my recent changes.